services/discountService.js

```diff
@@ -1,11 +1,14 @@
 function DiscountService() {}
 
 DiscountService.prototype.getDiscount = function (price) {
+    let target = -5;
+    let num = 3;
+    target = -num;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "target" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=javascript%3AS1854)